htdocs/product/inventory/inventory.php

```diff
@@ -844,6 +862,7 @@ function barcodeserialforproduct(tabproduct,index,element,barcodeproductqty,sele
 			console.log("Clear all values");
 			disablebuttonmakemovementandclose();
 			jQuery(".realqty").val("");
+			jQuery(".realqty").trigger("change");
```

Comment:
if the val is changd by jquery with jQuery(".realqty").val(""); the trigger change dont trigger but we need to trigger it to update calc of pmp and real value

Reply:
Why do we need to update real value ? Do you mean that with your interface, you can
Edit the real value and it autofill the real pmp or, at user choice, edit the real pmp and it update the real value ?
For me use must only edit the pmp, so it needs to update the real value of stock, but why the field real value is an "input field" ?

Reply:
Because some company can put the real value and it will update the pmp. Take a look at the video i linked  https://github.com/Dolibarr/dolibarr/pull/20135#issue-1145895475

Reply:
Looks strange to me in this direction but why not.